Marlin/Configuration.h

```diff
@@ -1662,9 +1662,24 @@
  *  - Change to green once print has finished
  *  - Turn off after the print has finished and the user has pushed a button
  */
-#if ENABLED(BLINKM) || ENABLED(RGB_LED) || ENABLED(RGBW_LED) || ENABLED(PCA9632) || ENABLED(NEOPIXEL_LED)
-  #define PRINTER_EVENT_LEDS
-#endif
+ #if ENABLED(BLINKM) || ENABLED(RGB_LED) || ENABLED(RGBW_LED) || ENABLED(PCA9632) || ENABLED(NEOPIXEL_LED)
+   #define PRINTER_EVENT_LEDS
+   #define LED_CONTROL_MENU                       // Uncomment to enable the LED control menu option.
```

Comment:
New features should be disabled by default.

Reply:
Yeah... sorry about that.